sparkle/scripts/python_model_preprocessing.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# coding: utf-8
+import os
+import glob
+import math
+
+import numpy as np
+import pandas as pd
+
+# point this to your local data directory (data not on Github)
+paths = glob.glob('new_data/*')
+paths = sorted(paths)[1:]
+paths
+
+
+
+def getMetaInfo(df):
+    
+    meta_df = df[["watchOnTwistHand", "n_pills"]].loc[[0]]
+    meta_df["pills_low"] = (meta_df["n_pills"] <= 10).astype(int)
+    
+    return meta_df
```

Comment:
Re-name function and add docstring to conform `with pep8`

```suggestion
def get_meta_info(df):
    """
    Create metadata df with columns = ["watchOnTwistHand", "n_pills", "pills_low"]
    """
    meta_df = df[["watchOnTwistHand", "n_pills"]].loc[[0]]
    # 1 if n_pills <= 10, 0 otherwise
    meta_df["pills_low"] = (meta_df["n_pills"] <= 10).astype(int)
    
    return meta_df
```